vochain/offchaindatahandler/offchaindatahandler.go

```diff
@@ -149,13 +165,7 @@ func (c *OffChainDataHandler) OnRevealKeys(pid []byte, priv string, txindex int3
 func (c *OffChainDataHandler) OnProcessStatusChange(pid []byte,
 	status models.ProcessStatus, txindex int32) {
 }
-func (c *OffChainDataHandler) OnSetAccount(addr []byte, account *vochain.Account) error {
-	return nil
-}
-func (c *OffChainDataHandler) OnTransferTokens(from, to []byte, amount uint64) error {
-	return nil
-}
+func (c *OffChainDataHandler) OnTransferTokens(from, to []byte, amount uint64) {}
```

Comment:
I remember to discuss this months ago and the idea is to introduce error in all the callbacks to handle possible failures. That is why I introduced those `error` in the functions and the idea was to modify the others at some point for having all of them returning error.
Let me know if you think we must never return `error` so I wont add them in the future.

Reply:
Yes, I think these event callbacks should not return errors (maybe only Commit). But I'm open to discuss it.

That's because the State eventListener is not gonna stop processing things because an arbitrary handler returns an error, as most it would print it, but then this can be probably better handled by the handler itself. 

So I think its more explicit for the handler developer to not include errors. So the developer knows that returning an error won't have any real effect.